tools/settings.py

```diff
@@ -23,7 +23,8 @@
     'MEMORY_GROWTH_GEOMETRIC_CAP',
     'GL_MAX_TEMP_BUFFER_SIZE',
     'MAXIMUM_MEMORY',
-    'DEFAULT_PTHREAD_STACK_SIZE'
+    'DEFAULT_PTHREAD_STACK_SIZE',
+    'ASYNCIFY_STACK_SIZE'
```

Comment:
Can you add a trailing comma here so future additions can just be single line change?

Reply:
Sure, done.